### ISSUE TITLE
todo's i think

### ISSUE THREAD

**Issue by ghtg** [brethower-k/binding-of-isaac-ai#2]
- bot cant shoot
- bot pathfinding is bad
- bot dont understand that shop items is not pedestal items
- bot cant avoid damage
- bot cant find a secret rooms
- bot dont destroying poops and fireplaces

bot tested on repentance with external items descriptions mod and 
{
local makeIsaacInvincible = false
local killAllEnemiesOnRoomStart = false
}
options
please update the bot, its very good project
good luck

**Comment by 2-X** [comment]
yup the bot just navigates the rooms, which was a huge problem to solve as-is because we had to write the tooling that controls isaac. i don't plan on bringing this project back to life unless i or someone else thinks of a smart way to integrate a training-based solution for fighting & item selection
